pyocd/target/builtin/target_ytm32b1ld0.py

```diff
@@ -0,0 +1,85 @@
+# pyOCD debugger
+# Copyright (c) 2006-2013 Arm Limited
+# Copyright (c) 2021 Chris Reed
```

Comment:
Please replace the copyright with your own on all new files.

You can also add your copyright below the existing copyrights on previously existing files that you've modified. Please see the [contributor guide](https://github.com/pyocd/pyOCD/blob/main/CONTRIBUTING.md) for details.

Reply:
Sure will update this section. Thanks for the comment.

Reply:
Just want to make sure you get the credit! 😄